src/main/anovos/feature_recommender/feature_recommendation.py

```diff
@@ -222,7 +240,7 @@ def feature_recommendation(
                     df_append = pd.DataFrame(
                         [
                             [
-                                df_user[desc_column].iloc[i],
+                                df_user[name_column].iloc[i],
                                 "Null",
```

Comment:
Null should be replaced with NA as discussed
<img width="903" alt="Screenshot 2022-02-24 at 6 46 09 PM" src="https://user-images.githubusercontent.com/92550461/155626413-bfa79494-9f59-4065-81a1-442e276cc787.png">
.